README.md

```diff
@@ -0,0 +1,95 @@
+# Etendo
+This is the development repository of Etendo. <br>
+Etendo is a platform to manage the business flows, adaptable to the needs of the companies. Our mission is creating an adaptable, flexible, composable and scalable software, able to grows without restrictions from the start and offers an intuitive, customizable and complete solution for the companies.
+We are developing an international ERP and a platform that supports business development and scalable growth for our partners and their customers.
+
+To more information visit [etendo.software](https://etendo.software)
+
+### Requirements
+In this section you can read the [System Requirements](https://docs.etendo.software/en/technical-documentation/etendo-environment/requirements-and-tools/requirements).
```

Comment:
I suggest that you write it like this: 'The following [link](https://docs.etendo.software/en/technical-documentation/etendo-environment/requirements-and-tools/requirements) contains information regarding the system's requirements'

Reply:
You can read about the system requirements in the [System Requirements](https://docs.etendo.software/en/technical-documentation/etendo-environment/requirements-and-tools/requirements) section of the Etendo Documentation.